.build/deploy.sh

```diff
@@ -4,4 +4,22 @@ IFS=$'\n\t'
 
 echo "🚀 Preparing to deploy..."
 
-echo "✅ Done"
+echo "🔑 Decrypting files..."
+
+gpg --quiet --batch --yes --decrypt --passphrase="${GPG_SECRET}" \
+    --output golo-dev-sign.asc .build/golo-dev-sign.asc.gpg
+
+gpg --quiet --batch --yes --decrypt --passphrase="${GPG_SECRET}" \
+    --output gradle.properties .build/gradle.properties.gpg
+
+gpg --fast-import --no-tty --batch --yes golo-dev-sign.asc
+
+echo "📦 Publishing..."
+
+./gradlew publish
+
+echo "🧹 Cleanup..."
+
+rm gradle.properties golo-dev-sign.asc
```

Comment:
Maybe add a trap to clean even if the script is aborted ?
Since you have `set -e`, if the `./gradlew publish` fails, the script stops without cleaning...

Reply:
How would you do that?

Reply:
`$ help trap` :wink:

Reply:
You're the expert, fancy a commit on this branch? 😉

Reply:
Done